lib/grp.gd

```diff
@@ -711,6 +718,7 @@ InstallTrueMethod( IsGroup, IsAlmostSimpleGroup );
 
 # a simple group is almost simple
 InstallTrueMethod( IsAlmostSimpleGroup, IsSimpleGroup );
```

Comment:
Could you also remove this bad implication? Then we could merge your PR before my PRs #3521 and #3525 , which I will rebase and adapt to use `IsNonabelianSimpleGroup`.
```suggestion
```

Reply:
On second thought, you can also leave it in and I'll remove it in my PRs. Whichever you prefer.

Reply:
I've just taken it out :-)